org.opent2t.sample.hub.superpopular/com.insteon.hub/js/thingTranslator.js

```diff
@@ -398,18 +412,22 @@ class Translator {
         return sleep(1000).then(() => {
             return this._makeRequest(this._commandPath + '/' + commandId, 'GET')
                 .then( (response) => {
-                    if(response.status === 'pending')
-                    {
-                        if(count > 1){
-                            return this._getCommandResponse(commandId, --count);
-                        } else {
-                            this._throwError('Internal Error - Insteon command timeout.');
-                        }
-                    } else {
-                        return response;
+                    switch(response.status){
+                        case 'pending':
+                            if(count > 1){
+                                return this._getCommandResponse(commandId, --count);
+                            } else {
+                                this._throwError('Internal Error - Insteon command timeout.');
+                            }
+                            break;
+                        case 'succeeded':
+                            return Promise.resolve(response);
+                        case 'failed':
+                        default:
+                            this._throwError( 'Internal Error - Insteon command failed for deviceId ' + response.command.device_id + '.');
```

Comment:
I believe that when a device is offline, that get_status will report "failed".  I see this happening now with some devices, and it causes getPlatforms to fail.